src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,10 +28,10 @@
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"), new Remark("")),
+                getTagSet("friends"), new Remark(""), getSampleSaleList()),
```

Comment:
I think setting sample sales and remark for more than half the contacts would be good, especially for sales list

Reply:
Added more sample sales data and remarks as requested!